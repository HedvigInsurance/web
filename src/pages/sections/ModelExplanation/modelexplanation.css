.ModelExplanation {
  margin-bottom: 2em;
  padding: 4em 0 0;
}

.ModelExplanation__text {
  text-align: center;
}

.ModelExplanation__text p {
  font-size: 14px;
  line-height: normal;
  white-space: pre-line;
}

.ModelExplanation__heading {
  color: #0f007a;
  font-family: "CircularStd", sans-serif;
  font-size: 28px;
  font-weight: 400;
  line-height: normal;
  margin: 0;
}

.ModelExplanation__paragraph {
  margin-bottom: 3em;
  max-width: 600px;
}

.ModelExplanation__icons {
  text-align: center;
}

.ModelExplanation__icon {
  padding: 15px 15px 0;
}

@media screen and (min-width: 1024px) {

  .ModelExplanation__heading {
    font-size: 40px;
  }

  .ModelExplanation__text p {
    font-size: 18px;
  }
}
