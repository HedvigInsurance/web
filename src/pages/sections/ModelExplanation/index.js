import React from 'react';
import Lazyload from 'react-lazyload';

import './modelexplanation.css';

const ModelExplanation = () => (
  <section className="pure-g pure-centered ModelExplanation">
    <div className="pure-u-1-1 pure-u-md-5-6">
      <div className="pure-g pure-centered ModelExplanation__text">
        <div className="pure-u-1-1">
          <h1 className="ModelExplanation__heading">För ett gott ändamål</h1>
          <p className="pure-u-5-6 ModelExplanation__paragraph">
            Hedvig betalar dig blixtsnabbt och har inget intresse av att säga nej, eftersom allt
            överskott skänks till ett gott ändamål. Du&nbsp;väljer&nbsp;vilket!
          </p>
        </div>
      </div>
      <div className="pure-g pure-centered ModelExplanation__icons">
        <div className="pure-u-1-1 pure-u-md-1-3 ModelExplanation__icon">
          <Lazyload height={166} offset={200}>
            <img
              width={166}
              height={166}
              src="/assets/icons/profil/personlig_info.svg"
              alt="Hedvig Logo"
            />
          </Lazyload>
          <p>Hedvig tar en fast avgift för att ge dig blixtsnabb&nbsp;service</p>
        </div>
        <div className="pure-u-1-1 pure-u-md-1-3 ModelExplanation__icon">
          <Lazyload height={166} offset={200}>
            <img
              width={166}
              height={166}
              src="/assets/icons/perils/mina_prylar/drulle.svg"
              alt="Skadad telefon"
            />
          </Lazyload>
          <p>Resten av din månadskostnad går till att täcka&nbsp;skador</p>
        </div>
        <div className="pure-u-1-1 pure-u-md-1-3 ModelExplanation__icon">
          <Lazyload height={166} offset={200}>
            <img width={166} height={166} src="/assets/icons/profil/valgorenhet.svg" alt="Hjärta" />
          </Lazyload>
          <p>
            Ett bra år med lite skador blir det pengar över, det skänker vi till ett
            gott&nbsp;ändamål
          </p>
        </div>
      </div>
    </div>
  </section>
);

export default ModelExplanation;
