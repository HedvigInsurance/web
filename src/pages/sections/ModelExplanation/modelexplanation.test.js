import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ModelExplanation from './index';

describe('<ModelExplanation />', () => {
  it('Should render correctly', () => {
    const component = shallow(<ModelExplanation />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
