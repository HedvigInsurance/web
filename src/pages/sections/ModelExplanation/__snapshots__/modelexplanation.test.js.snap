// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModelExplanation /> Should render correctly 1`] = `
<section
  className="pure-g pure-centered ModelExplanation"
>
  <div
    className="pure-u-1-1 pure-u-md-5-6"
  >
    <div
      className="pure-g pure-centered ModelExplanation__text"
    >
      <div
        className="pure-u-1-1"
      >
        <h1
          className="ModelExplanation__heading"
        >
          För ett gott ändamål
        </h1>
        <p
          className="pure-u-5-6 ModelExplanation__paragraph"
        >
          Hedvig betalar dig blixtsnabbt och har inget intresse av att säga nej, eftersom allt överskott skänks till ett gott ändamål. Du väljer vilket!
        </p>
      </div>
    </div>
    <div
      className="pure-g pure-centered ModelExplanation__icons"
    >
      <div
        className="pure-u-1-1 pure-u-md-1-3 ModelExplanation__icon"
      >
        <LazyLoad
          height={166}
          offset={200}
          once={false}
          overflow={false}
          resize={false}
          scroll={true}
          unmountIfInvisible={false}
        >
          <img
            alt="Hedvig Logo"
            height={166}
            src="/assets/icons/profil/personlig_info.svg"
            width={166}
          />
        </LazyLoad>
        <p>
          Hedvig tar en fast avgift för att ge dig blixtsnabb service
        </p>
      </div>
      <div
        className="pure-u-1-1 pure-u-md-1-3 ModelExplanation__icon"
      >
        <LazyLoad
          height={166}
          offset={200}
          once={false}
          overflow={false}
          resize={false}
          scroll={true}
          unmountIfInvisible={false}
        >
          <img
            alt="Skadad telefon"
            height={166}
            src="/assets/icons/perils/mina_prylar/drulle.svg"
            width={166}
          />
        </LazyLoad>
        <p>
          Resten av din månadskostnad går till att täcka skador
        </p>
      </div>
      <div
        className="pure-u-1-1 pure-u-md-1-3 ModelExplanation__icon"
      >
        <LazyLoad
          height={166}
          offset={200}
          once={false}
          overflow={false}
          resize={false}
          scroll={true}
          unmountIfInvisible={false}
        >
          <img
            alt="Hjärta"
            height={166}
            src="/assets/icons/profil/valgorenhet.svg"
            width={166}
          />
        </LazyLoad>
        <p>
          Ett bra år med lite skador blir det pengar över, det skänker vi till ett gott ändamål
        </p>
      </div>
    </div>
  </div>
</section>
`;
